Fallout2/Fallout1in2/.gitignore

```diff
@@ -6,3 +6,5 @@ scr*.bmp
 screen/
 sfall-log.txt
 worldmap.dat
+master.dat
+critter.dat
```

Comment:
You're not supposed to have master/critter in this directory - .cfg points to `..\file.dat` anyway.

Reply:
Put it there once to populate for the ProtoManager, and then set delete Cache to false